src/typed-method-types/workflows/triggers/inputs.ts

```diff
@@ -10,12 +10,27 @@ export type InputParameterSchema = {
   required: (string | number)[];
 };
 
-/** The structure that must be provided to the workflow input to pass a value */
-type WorkflowInput = {
+interface BaseWorkflowInputs {
+  /** @description The actual value passed to the Workflow's input parameters */
+  value?: unknown;
+  /** @description Set to `true` if this value should be dynamically inputted by the user at runtime */
+  customizable_value?: true;
```

Comment:
I think we want this to be `customizable` actually and not `customizable_value` so it matches the JSON schema https://slack-github.com/slack/webapp/blob/7a0ae1b8b7580be91690c18db97cffb72541112d/api_metadata/schemas/slack-functions/value-template.json#L66-L77